src/contracts/atlas/SafetyLocks.sol

```diff
@@ -12,20 +12,21 @@ import "../types/EscrowTypes.sol";
 import "../types/LockTypes.sol";
 
 import { Storage } from "./Storage.sol";
-import { FastLaneErrorsEvents } from "../types/Emissions.sol";
+import { FastLaneErrorsEvents, AtlasEvents } from "../types/Emissions.sol";
```

Comment:
Nice, I like this errors and events split. Will focus on the surcharge stuff in this PR, but I've made a branch I can maybe merge into your other events PR for the split stuff.